eosio/create_accounts.sh

```diff
@@ -1,15 +1,11 @@
 #!/usr/bin/env bash
+# This script is USED ONLY FOR TESTING. accounts for production are being created maunally.
 # Create and fund a test accounts if needed.
 
-
 . env.sh
 
-CYAN='\033[1;36m'
-GREEN='\033[0;32m'
-NC='\033[0m'
-
 name_maker() {
-    shuf -zern12 {1..5} {a..z} {a..z} {a..z} | tr -d '\0'
+    echo "$(shuf -zern1 {a..z} | tr -d '\0')""$(shuf -zern11 {1..5} {a..z} {a..z} | tr -d '\0')"
```

Comment:
Nitpick, but you do not need two sets of double-quotes here.

```sh
# These two are identical
"$(shuf -zern1 {a..z} | tr -d '\0')""$(shuf -zern11 {1..5} {a..z} {a..z} | tr -d '\0')"
"$(shuf -zern1 {a..z} | tr -d '\0')$(shuf -zern11 {1..5} {a..z} {a..z} | tr -d '\0')"
```

Reply:
done